libs/components/split-view/src/lib/modules/split-view/split-view-drawer.component.ts

```diff
@@ -138,7 +134,9 @@ export class SkySplitViewDrawerComponent
   }
 
   public ngAfterViewInit(): void {
-    this.#updateBreakpoint();
+    this.#mediaQuerySvc.observe(this.drawerRef, {
```

Comment:
One you make `drawerRef` a signal, you can use an effect to start observing:

```
constructor() {
  effect(() => {
    const drawerRef = this.#drawerRef();

    if (drawerRef) {
      this.#mediaQuerySvc.observe(drawerRef, { updateResponsiveClasses: true });          
    }
  });
}
```

Or if you want similar behavior as `ngAfterViewInit()`, you can use `afterNextRender()` instead:

```
constructor() {
  afterNextRender({
    read: () => this.#mediaQuerySvc.observe(this.drawerRef(), { updateResponsiveClasses: true })
  });
}
```